apps/web/src/pages/Profile/AccountSettings/AccountSettingsPage.tsx

```diff
@@ -0,0 +1,273 @@
+                {featureFlags.notifications && (
+                  <TableOfContents.Section
+                    id={sections.notificationSettings.id}
+                    data-h2-padding-top="base(x3)"
+                  >
+                    <TableOfContents.Heading
+                      size="h3"
+                      icon={Cog8ToothIcon}
+                      color="secondary"
+                      data-h2-margin="base(0, 0, x1, 0)"
+                    >
+                      {sections.notificationSettings.title}
+                    </TableOfContents.Heading>
+                    <p data-h2-margin="base(0, 0, x1, 0)">
+                      {intl.formatMessage({
+                        defaultMessage:
+                          "The settings provided in this section allow you to control when you receive notifications and how they are delivered. Email notifications are delivered to the email provided on your profile, while in-app notifications are delivered to the notification pane found in the main menu next to your name.",
+                        id: "hnyf1f",
+                        description:
+                          "Subtitle for notification settings section on account settings page.",
+                      })}
+                    </p>
+                    <NotificationSettings
+                      ignoredEmailNotifications={ignoredEmailNotifications}
+                      ignoredInAppNotifications={ignoredInAppNotifications}
+                    />
+                  </TableOfContents.Section>
+                )}
+                <TableOfContents.Section
+                  id={sections.recruitmentAvailability.id}
+                  data-h2-padding-top="base(x3)"
+                >
+                  <TableOfContents.Heading
+                    size="h3"
+                    icon={IdentificationIcon}
+                    color="tertiary"
+                    data-h2-margin="base(0, 0, x1, 0)"
+                  >
+                    {sections.recruitmentAvailability.title}
+                  </TableOfContents.Heading>
+                  <p data-h2-margin="base(0, 0, x1, 0)">
+                    {intl.formatMessage(
+                      {
+                        defaultMessage: `By applying to a talent pool on the platform, you agree to receive notifications about related potential employment opportunities. You can disable these notifications using the "availability" controls below, or by reviewing your <link>current recruitment processes on your career timeline</link>.`,
+                        id: "WcEkbD",
+                        description:
+                          "Subtitle for recruitment availability section on account settings page.",
+                      },
+                      {
+                        link: (chunks: React.ReactNode) =>
+                          inlineLink(paths.profileAndApplications(), chunks),
+                      },
+                    )}
```

Comment:
@substrae Any comments on this?

Reply:
I think the original intent here wasn't to _enable_ notifications about potential related opportunities, but more that if you had those notification types turned on, you were opting in to be contacted about related jobs in those pools.

Reply:
Maybe something more like this: `When you are admitted into a talent recruitment process, you agree to receive notifications about potential opportunities that meet the criteria of the process. These notifications will only be sent if you've enabled "Job alert" notifications in the "Notification settings" section of this page. You can also disable these notifications on a per process basis using the controls found on the cards provided.`

Reply:
Makes me wonder if we need a "recruitment" category for notifications though @tristan-orourke. Job alerts are different than being headhunted.

Reply:
I figured this fell squarely in the "Application Updates" category of notifications, since (at least currently) any job offer like this is explicitly a response to an application.

Further thoughts about CASL:
- If I applied to a job opportunity, I would be upset to NOT be notified of any responses to my application
- Under subsection 10(10)(e), an application seems like sufficient implied consent to receive notifications/responses about the application, at least for 6 months:
> an inquiry or application, within the six-month period immediately before the day on which the message was sent, made by the person to whom the message is sent to any of those other persons, in respect of anything mentioned in any of paragraphs (a) to (c).
- **Technically, legally, CASL does not apply to us.** Our messages are not commercial activity. And explicitly, **CASL does not apply to the activities of the federal, provincial and territorial governments**. https://crtc.gc.ca/eng/com500/faq500.htm
- That's not to say we shouldn't respect our users preferences or avoid spamming them. But we don't need to worry about legal liabilities.

Reply:
okay, I think the updated text @substrae wrote makes sense. I also misread it a bit  thinking the notifications were only tangentially related but after reading the comments I understand now that it is updates about specific opportunities you applied for 😅

Reply:
Okay 👍 Fixed by [61f344c](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/61f344c2f154f13694c407ea4eb87a99fbbfec57)